etc/profile-a-l/audacity.profile

```diff
@@ -36,7 +36,7 @@ protocol unix,inet
 seccomp
 tracelog
 
-private-bin audacity
```

Comment:
As per conversation, let's fix audacity by dropping apparmor and keep private-bin intact.

Reply:
How now, brown cow?